tests/purs/failing/ClassHeadNoVTA6.out

```diff
@@ -0,0 +1,35 @@
+Error found:
+in module [33mMain[0m
+at tests/purs/failing/ClassHeadNoVTA6.purs:8:9 - 8:17 (line 8, column 9 - line 8, column 17)
+
+  No type class instance was found for
+  [33m               [0m
+  [33m  Main.Multi t0[0m
+  [33m             t1[0m
+  [33m             t2[0m
+  [33m             t3[0m
+  [33m             t4[0m
+  [33m             t5[0m
+  [33m               [0m
+  The instance head contains unknown type variables.
+
+  Note: The following type class members found in the expression require visible type applications to be unambiguous (e.g. [33mtyClassMember @Int[0m).
+    [33mMain.useMulti[0m
+      b, c, e
```

Comment:
I was taking a look at this again. The type class is defined as this:
```purs
class Multi a b c d e f | a c -> d f, b c -> a d where
  useMulti :: Int
```
So, isn't there at least two ways to call `useMulti` with the least number of variables?
- `b c -> a d`: once `b` and `c` are known, we also know `a` and `d`. I guess we also know what `f` is via the `a c -> d f` fun dep? So, we only need to specify what `b`, `c`, and `e` are via `useMulti @_ @b @c @_ @d @_`
- `a c -> d f`: once `a` and `c` are known, then so is `d` and `f`. However, we still need to indicate what `b` and `e` are. Thus, we need to specify `a`, `b`, `c`, and `e` here via `useMulti @a @b @c @_ @e @_`. 

However, a set of `a, b, c, e` is missing from the above message. Should that be the case? Or should this test change to better reflect multiple sets?

Reply:
So, this is correct. While the first fun dep is usable, it still requires `b`, `c`, and `e` to be specified. But if `b`, `c`, and `e` are specified, we can use the second fun dep to calculate `a` and then the first fun dep to calculate `f`, such that only `b`, `c`, and `e` are needed. So the `a c -> d f` fun dep is not pointless, one would never use it primarily to find a usage's corresponding instance.

Reply:
With that being said, the changelog should be updated to use a different example where one of the sets of variables needs to be provided, such as the `ClassHeadNoVTA5.purs`.